wealth_dynamics.md

```diff
@@ -3,7 +3,10 @@
 In addition to what's in Anaconda, this lecture will need the following
 libraries:
 
-```ipython
+```{code-block} python
```

Comment:
Thanks @AakashGfude .  Should this be `ipython` rather than `python`? Shell commands with `!` require IPython...

Reply:
Thanks, @jstac for highlighting this. I will change it to `ipython` .